doc/htmldoc/installation/user.rst

```diff
@@ -32,6 +32,8 @@ We have live media (.ova) if you want to run NEST in a virtual machine.
 Linux |linux|
```

Comment:
Is this file still linked?

Reply:
Yes, just not the top reference, subsections are referenced